### PR TITLE
[CodeQualityStrict] Enable MoveOutMethodCallInsideIfConditionRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: rules/doctrine-gedmo-to-knplabs/src/Rector/Class_/LoggableBehaviorRector.php [hunk 1]
```diff
@@ -95,8 +95,9 @@ public function refactor(Node $node): ?Node
         if ($classPhpDocInfo === null) {
             return null;
         }
+        $class = $classPhpDocInfo->hasByType(LoggableTagValueNode::class);
```
Comment (samsonasik) [inline rules/doctrine-gedmo-to-knplabs/src/Rector/Class_/LoggableBehaviorRector.php:98]: `::class` suffix arg value seems need to be handled

Comment (samsonasik) [reply rules/doctrine-gedmo-to-knplabs/src/Rector/Class_/LoggableBehaviorRector.php]: fixed.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by samsonasik** [comment]
rebased.

**Comment by TomasVotruba** [comment]
Thank you :clap:  :+1:

**Pull request merged**